qa/environment.yml

```diff
@@ -0,0 +1,31 @@
+name: covid19-qa
+channels:
+  - defaults
+  - pytorch
+dependencies:
+  - pip:
+    - transformers==2.8.0
+    - elasticsearch==7.6.0
+    - Werkzeug==0.16.1
+    - unidecode==1.1.1
+    - bcrypt==3.1.7
+    - Flask==1.1.2
+    - flask-restx==0.2.0
+    - flask-accepts==0.16.5
+    - Flask-Bcrypt==0.7.1
+    - flask-marshmallow==0.11.0
+    - Flask-Migrate==2.5.2
+    - PyJWT==1.7.1
+    - marshmallow==3.4.0
+    - gunicorn==20.0.4
+    - requests==2.23.0
+    - flask_cors==3.0.8
+    - marshmallow-dataclass==7.5.0
```

Comment:
This is the part I don't like much about merging the containers. Maybe it's better to keep them separate, with the QA one only with Flask.

Reply:
Maybe we can avoid stating the transitive dependencies here. Unfortunately, conda dependency management sucks, and it's a PITA to keep all the deps by hand. So I think it's more practical if we keep the ones we directly use.

Reply:
I change it in the last commit. Now the qa server only exposes a single flask endpoint. All the complexity is in the API container.